### PR TITLE
Updgrade icon fonts to SVG

### PR DESCRIPTION
**Pull request by dskamiotis**
## Why are you doing this?

References and use of src-fonts are still being used in AR and need to be upgraded.
This is part of a several sub tasks to deprecate its use.

Currently pullquotes are using icons which cause issues when rendering in certain browser and ios/android versions, with strange fallback icons when not loading (square box).

Issues regarding Icons and how SVg are more desirable:

Icon Accessibility: do not support screen readers, SVG's, however are accessible to a screen reader (they are treated as images)
Font icons are not as scalable as they are font sized as opposed to vector based.
In terms of performance icons can increase HTTP server requests.for caching
Icons are vulnerable to anti-aliasing techniques causing blurring
In addition svg's allow for a high degree of modifications without triggering additional server requests so can have a performance advantage

Please see: https://github.com/guardian/apps-rendering/pull/677

## Changes

- Upgraded Blockquote to SVG with customised styles to have consistent positioning and size
- Upgraded read me arrow in renderer to be svg with updated styling
- Eliminated `@font-Face` from code base
- Deleted `icons from 'styles'`
- Eliminated  `font-family: 'Guardian Icons'`

## Screenshots

| Renderer Read More | BlockQuote |
| --- | --- |
|  |  |

### CODE REVIEW

File: src/components/blockquote.tsx [hunk 1]
```diff
@@ -16,31 +17,37 @@ interface Props {
...
-    position: relative;
-    margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
-    padding: 0 ${remSpace[6]};
+const styles = (format: Format): SerializedStyles => {
+    const { kicker } = getPillarStyles(format.pillar);
+    return css`
+        font-style: italic;
+        position: relative;
+        margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
+        padding: 0 ${remSpace[5]};
```
Comment (webb04) [inline src/components/blockquote.tsx:26]: I think there was another element that used `padding: 0 ${remSpace[6]};` so the blockquote lined up

File: src/components/blockquote.tsx [hunk 2]
```diff
@@ -16,31 +17,37 @@ interface Props {
...
+
+        svg {
```
Comment (webb04) [inline src/components/blockquote.tsx:28]: Could we use the same space formatting used in this file?

**Review by webb04 — COMMENTED** [review]
Good work! just a couple of small things

File: src/components/blockquote.tsx (hunk 1 above)
Comment (dskamiotis) [reply src/components/blockquote.tsx:26]: correct I was playing around with that and forgot to change back

File: src/components/blockquote.tsx (hunk 2 above)
Comment (dskamiotis) [reply src/components/blockquote.tsx:28]: 👍

File: src/components/blockquote.tsx [hunk 3]
```diff
@@ -16,31 +17,37 @@ interface Props {
...
+                height: 62.4px;
+                margin-left: -9.2625px;
+                margin-bottom: -12px;
+                margin-top: 4.4px;
```
Comment (JamieB-gu) [inline src/components/blockquote.tsx:32]: Regarding the use of pixel values here, what happens if you scale the page up and down (`cmd +` and `cmd -` in a browser)?

File: src/renderer.ts [hunk 4]
```diff
@@ -317,19 +318,16 @@ const richLinkStyles = (format: Format): SerializedStyles => {
...
-        button::before {
-            ${icons}
-            content: '\\e005';
+        svg{
+            width: 1.0625rem;
             border-radius: 100%;
             border: solid 1px ${neutral[7]};
-            font-size: 12px;
-            padding: 3px 6px 4px 6px;
+            padding: 4px;
```
Comment (JamieB-gu) [inline src/renderer.ts:327]: Same question about pixels as above.

**Review by JamieB-gu — COMMENTED** [review]
Nice!

Couple questions about scaling.

**Review by webb04 — APPROVED** [review]

**Pull request merged**